gadget.yaml

```diff
@@ -19,16 +19,21 @@ volumes:
             target: EFI/boot/bootx64.efi
       - # Recovery partition
         type: E3C9E316-0B5C-4DB8-817D-F92DF00215AE
-        size: 8192000000
-        offset: 250000384
+        size: 12884901888
+      - name: ubuntu-seed
+        role: system-seed
+        filesystem: ext4
+        type: 0FC63DAF-8483-4772-8E79-3D69D8477DE4
+        size: 1200M
```

Comment:
having partition ubuntu-seed as ext4 is unusual. And having it as not ESP is unusual. I am not sure if it is valid to have ubuntu-seed with a role system-seed, whilst ESP is marked as system-seed-null.

Has this been checked with snapd team? @alfonsosanchezbeato